test/sortitionTreeTest.js

```diff
@@ -19,86 +19,122 @@ describe("SortitionTree", () => {
 
   describe("setLeaf", async () => {
     context("when one leaf is set", () => {
-      beforeEach(async () => {
-        const weight1 = 0x1234
-        const position1 = parseInt("00123456", 8)
-
-        const leaf = await sortition.toLeaf(alice.address, weight1)
-        await sortition.publicSetLeaf(position1, leaf, weight1)
-      })
-
       it("should return correct value for the tree", async () => {
```

Comment:
... fot the tree with a leaf in the first slot?

Reply:
[415e151](https://github.com/keep-network/sortition-pools/pull/180/commits/415e151be37db3bd0ae3202a20c29092ff7ff61f)